### PR TITLE
[openfhe_julia_int128] initialize and [openfhe_julia] update

### PR DESCRIPTION
**Pull request by ArseniyKholod**

### CODE REVIEW

**Comment by ArseniyKholod** [comment]
I have opened an issue to llvm about the typeinfo for __int128 problem: https://github.com/llvm/llvm-project/issues/119608.
For now, I disable Apple compilations for 128-bit version.

File: O/openfhe_julia/common.jl [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    
+    # For MacOS additional linker flag is required to link typeid(__int128)
+    if [[ "$target" == *-apple-darwin* ]]; then
+        export LDFLAGS="-lc++abi"
+    fi
+    export LDFLAGS="-lc++abi"
+    # For MacOS additional linker flag is required to link typeid(__int128)
+    if [[ "$target" != *-apple-darwin* ]]; then
+        export LDFLAGS=""
+    fi
```
Comment (sloede) [inline O/openfhe_julia/common.jl:51]: @ArseniyKholod this seems wrong?

Comment (ArseniyKholod) [reply O/openfhe_julia/common.jl]: Now it is actually fails only on x86_64, but aarch64 is OK

Comment (ArseniyKholod) [reply O/openfhe_julia/common.jl]: And I tested only aarch86 in our repo

**Review by sloede — APPROVED** [review]
LGTM! And thanks a lot for keeping at it until you found a solution for the macOS issues 💪

**Pull request merged**